gapic/templates/examples/feature_fragments.j2

```diff
@@ -25,44 +25,69 @@ There is a little, but not enough for it to be important because
 
 {# response handling macros #}
 
+{% macro sample_header(fileHeader, sample, callingForm) %}
+#
+# DO NOT EDIT! This is a generated sample ("{{ callingForm }}",  "{{ sample.id }}")
+#
+# To install the latest published package dependency, execute the following:
+#   pip3 install {{ sample.package_name }}
+{% endmacro %}
+
+{% macro printInputParams(requests) %}
+{% with input_parameters = [] %}
+{% for request in requests %}
+  {% for element in request.body %}
+    {% if "input_parameter" in element %}
+      {% do input_parameters.append(element["input_parameter"]) %}
+    {% endif %}
+  {% endfor %}
+{% endfor %}
+{{ input_parameters|join(", ") -}}
+{% endwith %}
+{% endmacro %}
+
 {% macro renderPrint(elts) %}
   {# First elment is a format string, remaining elements are the format string parameters #}
   {# Validating that the number of format params equals #}
   {# the number of remaining params is handled by real python code #}
   {% with fmtStr = ('\"' + elts[0] + '\"') |replace("%s", "{}") %}
-print({{ ([fmtStr] + elts[1:])|join(', ') }})
-  {% endwith -%}
+  {% with params = elts[1:]|map("coerce_variable_name")|list %}
```

Comment:
Just curious: what does `elts` stand for? Is it a standard abbreviation?

Reply:
It's intended to be short for 'elements'. I've seen other people use it a few places.

Reply:
Ah! I've not seen it before. I've seen `elems`. Another abbrev added to my data banks.